src/07-projects/materials.apib

```diff
@@ -0,0 +1,61 @@
+## Materials [/materials]
+
+Materials are added to a project and they affect the profitability. This information is only available for superusers and project decision makers.
+
+*Required scopes: `projects`*
+
+### materials.list [GET /materials.list]
+
+Get a list of project materials.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string)
+                + project (object)
+                    + type: `project` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + name: `Software license` (string)
+                + unit (object)
+                    + cost (object)
+                        + Include Money
+                        + tax: `excluding` (enum[string])
+                            + Members
+                                + excluding
+                    + price (object)
+                        + Include Money
+                        + tax: `excluding` (enum[string])
+                            + Members
+                                + excluding
+                + quantity: `3.0` (number)
+                + total (object)
+                    + cost (object)
+                        + Include Money
+                        + tax: `excluding` (enum[string])
+                            + Members
+                                + excluding
+                    + price (object)
+                        + Include Money
+                        + tax: `excluding` (enum[string])
+                            + Members
+                                + excluding
+                + billable: true (boolean)
+                + milestone (object)
+                    + type: `milestone` (string)
+                    + id: `0709fc3e-e918-00d5-a113-ba00a33102ca`
```

Comment:
PP: specify that this is a string
(same for supplier id)

Reply:
^ and department. String seems to be the default value, but I agree it's better to make it explicit.